CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 
 ## vN.N.N
 * issue #1035: Add Spotify Embed Block
+* Issue: Add Imgur Post Block
+
+
+## v0.2.16
+* Fix mediaLive reducer bug where endpoints can be duplicated.
```

Comment:
@jalopez13 somethings wonky here, this is already in release-next changelog

Reply:
@efjacobson 

Pulled latest release-next and updated this entry.